tests/compiler/circuits/constant_member.leo

```diff
@@ -0,0 +1,16 @@
+/*
+namespace: Compile
+expectation: Pass
+input_file: input/dummy.in
+*/
+
+circuit Foo {
+    static const x: u32 = 4u32;
+    static const y: u32 = 23;
+}
+
+function main(y: bool) -> bool {
+    let z = Foo::x + 32;
```

Comment:
Could you add tests (?) where we assign to:
- `let f = Foo {}; f.x = 0;`
- `Foo::y = 0;`